### PR TITLE
test2

### PR DESCRIPTION
**Pull request by s709t**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  acl    = "private"
+  tags = {
+    Name        = var.bucket
+    Environment = "Dev"
+  }
+}
+resource "aws_s3_bucket_object" "upload_files" {
+  bucket = aws_s3_bucket.new_bucket.bucket
+  for_each = fileset(path.module, "*.txt")
+  key    = each.value
```
Comment (diegows) [inline main.tf]: You can do this only with Terraform, no Python scripting is required. Terraform provides functions for timestamps.

File: run.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+python3 files.py
```
Comment (diegows) [inline run.sh]: Check previous comment.

File: main.tf (hunk 1 above)
Comment (s709t) [reply main.tf]: The changes were applied

File: run.sh (hunk 2 above)
Comment (s709t) [reply run.sh]: File removed

**Pull request merged**